src/content/updates/v10-migration/overview.mdx

```diff
@@ -1,46 +1,47 @@
 ---
-title: v10 Migration
+title: Migrate to v10
 tabs: ['Overview', 'Designers', 'Developers']
 ---
 
+### It's Carbon's biggest overhaul yet. New layouts, new colors, new guidelines, and a whole new website. Here's everything you need to migrate [from v9](https://v9.carbondesignsystem.com/).
```

Comment:
When I first come to the page I see a large link to v9. Since this is an h3 is it appropriate to have as a link? We may not need it now that there's a large card just below it giving more context about where v9 went.

Reply:
Yeah I think now that there's a tile we can take out this hyperlink

Reply:
Agreed!